packages/order-utils/src/hash.ts

```diff
@@ -0,0 +1,77 @@
+import { schemas, SchemaValidator } from '@0xproject/json-schemas';
```

Comment:
Let's rename this module `orderHash`? Feel it's more precise... Esp. since we will also have ZeroExTransactionHash in V2 (https://github.com/0xProject/0x-monorepo/pull/498)